lib/src/js/process_elements.sh

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env bash
 
-node packages/custom_element_apigen/src/js/analyze.js $*
+X=$PWD
```

Comment:
How about just eliminating this script all together? And instead just use `Process.run('node', [$scriptPath, inputPath])`? (or something like that).